redis/src/cluster_routing.rs

```diff
@@ -159,7 +246,7 @@ impl RoutingInfo {
                 Some(Aggregate(AggregateOp::Sum))
             }
 
-            b"MSETNX" | b"WAIT" => Some(Aggregate(AggregateOp::Min)),
+            b"WAIT" => Some(Aggregate(AggregateOp::Min)),
```

Comment:
so whats the response policy of MSETNX?

Reply:
https://github.com/redis/redis/pull/12417
it shouldn't have a request or response type, since it can't be split. It's atomic.